sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -199,6 +208,9 @@ export interface CoreSubjectAlternativeNames {
 export interface CreateCertificateOptions extends CertificateProperties, coreHttp.OperationOptions {
 }
 
+// @public
+export type CreateCertificatePollerLike = PollerLike_2<PollOperationState<KeyVaultCertificateWithPolicy>, KeyVaultCertificateWithPolicy>;
```

Comment:
Example of the type being exposed in the public interface

Reply:
Do you mean PollerLike? this PollerLike needs to be exposed since it's not available through core-lro (it's our custom one).

Reply:
PollerLike_2

Reply:
Fixed now.